docs/maintain/iota-core/2.0/docs/how_tos/validator_setup.md

```diff
@@ -21,83 +21,300 @@ import LinuxCommands from '@site/docs/_admonitions/_linux-commands.md';
+
+### 4. Begin Staking
+
+You can use the following command to start staking:
+
+```
+
+./wallet begin-staking {Account ID} {Stake Amount} {Fixed Cost}
+
+```
+
+You can verify you successfully started staking by running the `sync` and `accounts` commands, copying the `Output ID` of the account, and then running `output {Output ID}`. You should be able to find the `StakingFeature` in within the `features` section. It should show the `Stake Amount` and `Fixed Cost` you just entered. It should look like this, but with the values you entered:
+
+```
+
 StakingFeature {
-    staked_amount: 1000000000,
-    fixed_cost: 1,
-    ...
+staked_amount: 1000000000,
+fixed_cost: 1,
+...
+
 ```
 
-The account is now registered as a validator. Now we need to setup the infrastructure to actually issue validation blocks to secure the network.
+The account is now registered as a validator. Next, you must set up the infrastructure to issue validation blocks to secure the network.
 
 ## Prepare the Validator Plugin
 
-1. Running a validator requires that you run a full iota-core node as well. Follow the steps outlined [here](./using_docker.md) to setup the node. Follow the next steps either after you've successfully setup the node or while setting it up, at your choice.
-2. The `.env` file needs to be modified in the following ways.
-3. Uncomment the `COMPOSE_PROFILES` line below the validator service.
-4. Set the `Account Address` from earlier as the `VALIDATOR_ACCOUNT_ADDR`.
-5. Set the `ed25519 private key` from earlier as the `VALIDATOR_PRV_KEY` (**with no** 0x prefix).
-6. In order to issue a candidacy announcement to the network the used account needs Mana. It must be _allotted_ to the account which can be done via the `allot-mana` command in the CLI wallet or via the developer tools in Firefly.
-6. When you start the docker containers, an `inx-validator` container should be started. Check the logs to see if everything is working with: `docker logs -f inx-validator`.
+Running a validator requires running a full IOTA-core node. You can follow the steps outlined in the [how to install using Docker guide](./using_docker.md) to setup the node. 
+
+Follow the next steps after successfully setting up the node or while setting it up at your choice.
+
+### 1. Update the `.env` File
+
+You must modify the `.env` file in the following ways:
+
+1. Uncomment the `COMPOSE_PROFILES` line below the validator service.
+2. Set the `Account Address` from earlier as the `VALIDATOR_ACCOUNT_ADDR`.
+3. Set the `ed25519 private key` from earlier as the `VALIDATOR_PRV_KEY` (**with no** 0x prefix).
+
+### 2. Allot Mana to the Account
+
+The selected account needs Mana to issue a candidacy announcement to the network. It must be _allotted_ to the account. You can do this using the  `allot-mana` command in the CLI wallet or the developer tools in Firefly.
+
+```bash
+./wallet allot-mana
```

Comment:
Thoralf is still fixing this command as it currently doesn't work reliably and we need to extend the instructions here a bit further once fixed (how much to allot, which isn't that easy to answer unfortunately).